contracts/token/ERC1155/extensions/ERC1155Supply.sol

```diff
@@ -57,9 +60,9 @@ abstract contract ERC1155Supply is ERC1155 {
         if (from == address(0)) {
             uint256 totalMintValue = 0;
             for (uint256 i = 0; i < ids.length; ++i) {
-                uint256 value = values[i];
+                uint256 value = values.unsafeMemoryAccess(i);
```

Comment:
This assumes `values.length == ids.length`. It is something that is checked in `ERC1155._update`, but that may be lost if another override of `_update` (between this and `ERC1155` in the linearized list or contracts) fails to call `super`.

I'm not too concerned about that.

Reply:
Well, if we merge that change it will not panic. 
This PR removes the check that would cause the panic to happen

Reply:
Right, got confused.
Then what do you think of adding that requirement in the fn natspec?

Reply:
Thought about it and concluded it's not necessary since it's already checked in ERC1155 and we're already warning people about overrides without calling `super`.

Solving